README.md

```diff
@@ -78,6 +82,20 @@ found in the `src/` directory.
     - stylesheets/
 ```
 
+`require('uswds')` will load all of Draft U.S. Web Design Standard's JavaScript onto the page. The uswds module itself does not export anything.
+
+The main Sass (SCSS) source file is here: 
+
+```
+node_modules/uswds/src/stylesheets/all.scss
+```
+
+The non-minified CSS that’s been precompiled is here:
```

Comment:
@maya @juliaelman 
I see, well for the `npm` instructions there is no way for them to used the minified code using `require('uswds');` They need to use the source files and the entry point is referenced in the `package.json` [here](https://github.com/18F/web-design-standards/blob/18f-pages-staging/package.json#L5). Minifiying the source when using `npm` is something that those specific developers would need to do on their own. _We could definitely add a sentence about that though._ Thoughts?